obal/data/modules/srpm.py

```diff
@@ -0,0 +1,89 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import commands
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir
+
+def run_command(command):
+    return commands.getstatusoutput(command)
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    command = "spectool --list-files %s" % spec_file
+    _status, sources = run_command(command)
+
+    source_files = []
+
+    for source in sources.split("\n"):
+        if 'http' not in source:
```

Comment:
This should be a `startswith()` to avoid issues with local files that have `http` in them (like `fix-http.patch`)